src/structures/game_session.ts

```diff
@@ -619,6 +612,62 @@ export default class GameSession extends Session {
         await super.updateOwner();
     }
 
+    async handleClipComponentInteraction(
+        round: GameRound,
+        interaction: Eris.ComponentInteraction<Eris.TextableChannel>,
+        messageContext: MessageContext,
+    ): Promise<boolean> {
+        if (
+            Date.now() - round.songStartedAt! <
```

Comment:
```
round.songStartedAt!
```
Is it possible for the clip button to be sent before the song starts and this is assigned a value? lets add a null check here just incase

Reply:
Is this conditional just saying "cant request new clip" before the first replay?

Reply:
The `clipGameRound.getReplayCount() < 1` was an afterthought, I can take it out